### ISSUE TITLE
Bitter missing glyphs

### ISSUE THREAD

**Issue by goodpixels** [google/fonts#25]
Polish 'ś' character does not correctly - although it is supported by this font.

**Comment by codeman38** [comment]
The version in Google Fonts is *really* old compared to the one that's available on Huerta Tipografica's site (http://huertatipografica.com/fonts/bitter-ht). Is there any chance that the Google version might be updated to match?

**Comment by AndydeCleyre** [comment]
More glyphs please!

Is there a better way to request this? This ticket has been left open for over a year.

Actually, every ticket in this repo that I care about has been left open for over a year . . .

**Comment by davelab6** [comment]
Commenting here is the best way :)

**Comment by dom1n1k** [comment]
Please update version of Bitter. Cyrillic subset is needed.

**Comment by thlinard** [comment]
The last available version is 2.110: https://github.com/solmatas/Bitter
There've been more recent developments, but not yet sync with git.

**Comment by laerm0** [comment]
Newest version is still 2.110 on github. I wonder if this project has been mothballed...

**Comment by davelab6** [comment]
It's actively maintained by @solmatas - I guess sol needs to push everything to GitHub if Thomas is right that the work shown on the development mailing list isn't there and then a review and pr by @m4rc1e

**Comment by thlinard** [comment]
No reaction to my last post: https://groups.google.com/d/msg/googlefonts-discuss/Z_BHvgSV7IE/a8r1qDYsAgAJ

**Comment by thlinard** [comment]
@solmatas just created a dev branch!

**Comment by solmatas** [comment]
I am so sorry for the delay.

I was waiting for the final review of the italic Cyrillic to upload everything together.
=S


.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*

Sol Matas
. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 

HUERTA TIPOGRÁFICA
sol@huertatipografica.com
huertatipografica.com
facebook.com/huertatipo

**Comment by thlinard** [comment]
On Sol Matas' own website, Bitter is presented as a family of 9 weights + italics, with Latin + Cyrillic + Devanagari support.
https://www.solmatas.com/#/bitter/

**Comment by laerm0** [comment]
I've always liked this family and would love to get the updates out. @solmatas I'd like to work on this at the start of next year. Is that good for you? Want to work together?

**Comment by thlinard** [comment]
I just noticed Bitter is on Adobe Fonts… in 6 styles!!
https://fonts.adobe.com/fonts/bitter

Thus, we have:

1. Bitter, 3 styles (Regular, Italic, Bold) on Google Fonts. Version 1.002.
2. Bitter, 6 styles (Regular, Bold, Black + italics) on Adobe Fonts. Version 1.3 (but 1.0 for the Blacks, which lack Cyrillic). It's a bit paradoxical that the best way to have a Google Fonts is to look for it at Adobe. 😁
3. Bitter, 4 styles (Regular, Bold + italics) on https://github.com/solmatas/Bitter. Version 2.110 (GF Cyrillic Pro + Vietnamese support).
4. Bitter, 9 styles on https://www.solmatas.com/#/bitter/. Unavailable.

What a mess for a font with so much potential!

Could at least the version of this repo be updated to version 2.110? It's the last one available anyway.

What do you think @laerm0?

**Comment by laerm0** [comment]
Indeed, I'll get going on this soon. Thanks for the reminder @thlinard. @solmatas, want to work on this together? I can handle it all on my end, I thought I'd offer.

**Comment by laerm0** [comment]
Huh! Interesting @thlinard. This will definitely be on my list for February. Thanks for the research. 👍

**Comment by thlinard** [comment]
You're welcome. 😀

Btw @laerm0, I found the sources of Adobe version. Here: https://github.com/huertatipografica/Kadwa/tree/master/otf

**Comment by laerm0** [comment]
Ah, excellent, thanks again!

**Comment by thlinard** [comment]
Hi @laerm0 

Sol Matas has just informed me that she had uploaded a new version, with "extended Latin support, Small caps, a wider set of figures and symbols, and extended Cyrillic support". But maybe you or @davelab6  have something to do with it? 😉

https://github.com/solmatas/BitterPro

Also, "A new weight distribution across the variables was applied". Maybe Marc won't like it. 😄

The version is 1.001, which is weird as the last release was 2.110. A version 3.000 would have been more judicious, IMO.

Other than that, the sources seem ready to make a VF version.

**Comment by thlinard** [comment]
It seems that development is now supported by TypeNetwork (Google Fonts contractor, I believe):

https://github.com/TypeNetwork/BitterPro

**Comment by thlinard** [comment]
Pull #2548: this issue can finally be closed!

**Comment by c02y** [comment]
> The version in Google Fonts, 1.001 from 2011, is _really_ old compared to the one that's available on Huerta Tipografica's site (http://huertatipografica.com/fonts/bitter-ht), which is 1.3 from 2013. The latter includes not only extended-Latin but also Cyrillic. Is there any chance that the Google version might be updated to match?

Why the fonts I downloaded from http://huertatipografica.com/fonts/bitter-ht are 1.010 from 2011?
